.travis.yml

```diff
@@ -7,6 +7,15 @@ os: osx
 # on Traivs.
 env:
   - TEST_DIR=com.google.code.gson-gson
+  - TEST_DIR=org.joda-joda-convert
+  - TEST_DIR=joda-time-joda-time
```

Comment:
yes that is indeed the ugly fully qualified name for this.

like apache, joda is not consistent with its group id